app/helpers/content_helper.rb

```diff
@@ -41,7 +41,13 @@ def html(html, new_tab_for_all_links: true)
         end
       end if request
 
-      doc.to_html.html_safe # rubocop:disable Rails/OutputSafety
+      if glossary == true
+        glossary = GlossaryHelper::DEFAULT_OPTIONS
+      end
+
+      link_glossary_terms(doc, glossary) if glossary
```

Comment:
By default, glossary terms are not linked more than once on the same page. You can override this by passing an options hash instead of true to `html`, e.g.

https://github.com/EFForg/sec/blob/387ad9c16130d283ae0ae1a42dedc53f45b84ffe/app/views/glossary/index.html.erb#L8